File: SMCDev/I love you, a little , a lot, passionately ... not at all.js
function howMuchILoveYou(nbPetals) {
  switch ((nbPetals - 1) % 6) {
    case 1:
      return "a little";
      break;
    case 2:
      return "a lot";
      break;
    case 3:
      return "passionately";
      break;
    case 4:
      return "madly";
      break;
    case 5:
      return "not at all";
      break;

    default:
      return "I love you";
      break;
  }
}

File: Ruby/I love you, a little , a lot, passionately ... not at all.rb
def how_much_i_love_you(nb_petals)
  case nb_petals % 6
  when 1
    return "I love you"
  when 2
    return "a little"
  when 3
    return "a lot"
  when 4
    return "passionately"
  when 5
    return "madly"
  when 0
    return "not at all"
  end
end
